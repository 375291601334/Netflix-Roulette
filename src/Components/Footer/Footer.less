footer {
  display: flex;
  padding: 20px 100px;
  justify-content: center;
}
