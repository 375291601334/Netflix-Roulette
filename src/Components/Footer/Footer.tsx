import React from 'react';
import PropTypes from 'prop-types';
import './Footer.less';

export function Footer(props) {
  return (
    <footer>{props.children}</footer>
  );
}

Footer.propTypes = {
  children: PropTypes.element.isRequired,
};
