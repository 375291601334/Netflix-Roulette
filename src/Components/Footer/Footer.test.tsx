import React from 'react';
import { render } from '@testing-library/react';
import { Footer } from './Footer';

describe('Footer', () => {
  test('render Footer', () => {
    const { baseElement } = render(<Footer><span>Logo</span></Footer>);
    expect(baseElement).toMatchSnapshot();
  });
});
