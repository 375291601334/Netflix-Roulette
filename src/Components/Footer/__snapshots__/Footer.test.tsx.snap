// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer render Footer 1`] = `
<body>
  <div>
    <footer>
      <span>
        Logo
      </span>
    </footer>
  </div>
</body>
`;
